### ISSUE TITLE
it can't display any data

### ISSUE THREAD

**Issue by bobzhanggmail** [rickbergfalk/sqlpad#504]
hi，
 I have installed sqlpad with docker. the service is ok after install.
 When i add a connection and add a query message,it can't display any data now.
 Docker logs sqlpad:
```
[session-file-store] will retry, error on last attempt: Error: ENOENT: no such file or directory, open '/var/lib/sqlpad/sessions/GuEwwaZ-s_V77gAWKErjD-NrkjYL8ot5.json'
[session-file-store] will retry, error on last attempt: Error: ENOENT: no such file or directory, open '/var/lib/sqlpad/sessions/GuEwwaZ-s_V77gAWKErjD-NrkjYL8ot5.json'
```

**Comment by bobzhanggmail** [comment]
My db is Mysql 5.7,
and os is Centos 7 X64,
Thank you!

**Comment by bobzhanggmail** [comment]
![1111](https://user-images.githubusercontent.com/9858448/71608232-9f046c00-2bba-11ea-9015-f1f9fb188940.png)

**Comment by rickbergfalk** [comment]
Thanks @bobzhanggmail -- I believe we have this fixed in #498. Will tag a release with fixes shortly

**Comment by rickbergfalk** [comment]
Should be fixed in 3.5.2 or latest. If that is not the case let me know and we can reopen ticket.
